src/components/Instructions.component.js

```diff
@@ -5,16 +5,38 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
-    return (
-      <div className="instructions">
-        Follow the instructions on the README to get started!
-        <img
-          className="instructions__logo"
-          src="https://uiuc.hack4impact.org/img/colored-logo.png"
-          alt="h4i logo"
-        />
-      </div>
-    )
+    const items = this.props.items
+    let newItems = []
+    for (let i = 0; i < items.length; i++) {
+      if (items[i].length <= 3) {
+        if (i % 2 == 0) {
+          let temp = items[i].toUpperCase()
+          newItems.push(temp + ' ')
+        } else {
+          newItems.push(items[i] + ' ')
+        }
+      }
+    }
+    if (this.props.shouldDisplayImage) {
+      return (
+        <div className="instructions">
+          Follow the instructions on the README to get started!
+          <img
+            className="instructions__logo"
+            src="https://uiuc.hack4impact.org/img/colored-logo.png"
+            alt="h4i logo"
+          />
+          <ul>{newItems}</ul>
+        </div>
+      )
+    } else {
+      return (
+        <div className="instructions">
+          Follow the instructions on the README to get started!
+          <ul>{newItems}</ul>
+        </div>
+      )
+    }
```

Comment:
You can get rid of this if-else statement and use conditional rendering. It would look something like this:
{this.props.shouldDisplayImage && ( /**JSX code**/)}

Here is a link for reference :) https://reactjs.org/docs/conditional-rendering.html